### PR TITLE
Change AutoStart type from bool to int

### PR DESCRIPTION
**Pull request by veenstrajelmer**
AutoStart type should be int instead of bool, since 2 is also a allowed value according to the manual: https://content.oss.deltares.nl/delft3d/manuals/D-Flow_FM_User_Manual.pdf#section.A.1

### CODE REVIEW

File: hydrolib/core/io/mdu/models.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ class General(INIGeneral):
     version: str = Field("1.2.94.66079M", alias="version")
     filetype: Literal["modelDef"] = Field("modelDef", alias="fileType")
     fileversion: str = Field("1.09", alias="fileVersion")
-    autostart: bool = Field(False, alias="autoStart")
+    autostart: int = Field(0, alias="autoStart")
```
Comment (priscavdsluis) [inline hydrolib/core/io/mdu/models.py:33]: We also like to use enums for limited options such as in this case. 
This comes with the advantage that we can also document the several options. For example that `0` is `no`, `1` = `autostart` and `2` = `autostartstop`. 

I _think_ (not sure) that we get also "free" validation on the values, for example that 3 or -1 is not valid.


```
class AutoStartOption(IntEnum):
    """
    Enum class containing the valid values for the AutoStart
    attribute in the [General][hydrolib.core.io.mdu.models.General] class.
    """
    no=0
    autostart=1
    autostartstop=2
```


```
autostart: AutoStartOption = Field(AutoStartOption.no, alias="autoStart")
```

**Review by priscavdsluis — CHANGES_REQUESTED** [review]
@veenstrajelmer, I reviewed the changes. Please let me know what you think.

**Comment by veenstrajelmer** [comment]
Great, looks good.

File: hydrolib/core/io/mdu/models.py [hunk 2]
```diff
@@ -662,7 +672,7 @@ class Processes(INIBasedModel):
     thetavertical: Optional[float] = Field(0.0, alias="ThetaVertical")
     dtprocesses: Optional[float] = Field(0.0, alias="DtProcesses")
     dtmassbalance: Optional[float] = Field(0.0, alias="DtMassBalance")
-    processfluxintegration: Optional[float] = Field(
+    processfluxintegration: Optional[ProcessFluxIntegration] = Field(
```
Comment (priscavdsluis) [inline hydrolib/core/io/mdu/models.py:675]: Good catch!

**Pull request merged**